services/text_to_speech/tts-utils.js

```diff
@@ -45,23 +49,25 @@ class TTSUtils {
   static initTTSService(req, sApikey, sUsername, sPassword, sEndpoint) {
     const endpoint = req.query.e ? req.query.e : sEndpoint;
 
+    let authSettings  = {};
     let serviceSettings = {
       url: endpoint,
       headers: {
         'User-Agent': pkg.name + '-' + pkg.version
       }};
 
     if (sApikey || req.query.key) {
-      serviceSettings.iam_apikey = sApikey ? sApikey : req.query.key;
+      authSettings.apikey = sApikey ? sApikey : req.query.key;
     } else {
-      serviceSettings.username = sUsername ? sUsername : req.query.un;
-      serviceSettings.password = sPassword ? sPassword : req.query.pwd;
+      authSettings.username = sUsername ? sUsername : req.query.un;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4764899)